src/fabric.c

```diff
@@ -437,79 +437,42 @@ static struct fi_provider *ofi_get_hook(const char *name)
 	return provider;
 }
 
-struct load_entry {
-	const char* provider_name;
-	int should_load;
-};
-
-#define DEFINE_LOAD_ENTRY(NAME, SHOULD_LOAD) \
-    { \
-        .provider_name = NAME, \
-        .should_load = SHOULD_LOAD \
-    }
-
 /* This is the default order that providers will be accessed when available.
  * This, in turn, sets the default ordering of fi_info's reported to the user.
  * However, ofi_reorder_info() may re-arrange the list based on hard-coded
  * criteria.
  */
 static void ofi_ordered_provs_init(void)
 {
-	struct load_entry ordered_load_list[] = {
-		DEFINE_LOAD_ENTRY("efa", (HAVE_EFA | HAVE_EFA_DL)),
-		DEFINE_LOAD_ENTRY("psm2", (HAVE_PSM2 | HAVE_PSM2_DL)),
-		DEFINE_LOAD_ENTRY("opx", (HAVE_OPX | HAVE_OPX_DL)),
-		DEFINE_LOAD_ENTRY("usnic", (HAVE_USNIC | HAVE_USNIC_DL)),
-		DEFINE_LOAD_ENTRY("gni", (HAVE_GNI | HAVE_GNI_DL)),
-		DEFINE_LOAD_ENTRY("bgq", (HAVE_BGQ | HAVE_BGQ_DL)),
-		DEFINE_LOAD_ENTRY("verbs", (HAVE_VERBS | HAVE_VERBS_DL)),
-		DEFINE_LOAD_ENTRY("netdir", 0),
-		DEFINE_LOAD_ENTRY("psm3", (HAVE_PSM3 | HAVE_PSM3_DL)),
-		DEFINE_LOAD_ENTRY("ucx", (HAVE_UCX | HAVE_UCX_DL)),
-		DEFINE_LOAD_ENTRY("ofi_rxm", (HAVE_RXM | HAVE_RXM_DL)),
-		DEFINE_LOAD_ENTRY("ofi_rxd", (HAVE_RXD | HAVE_RXD_DL)),
```

Comment:
Do all the windows marcos need to be removed?

Reply:
It's fine to leave them there.